macros/AddonSidebar.ejs

```diff
@@ -71,13 +71,15 @@ function currentPageIsUnder(root) {
         <li><a href="<%=baseURL%>WebExtensions/Comparison_with_the_Add-on_SDK">Comparison with the Add-on SDK</a></li>
         <li><a href="<%=baseURL%>WebExtensions/Comparison_with_XUL_XPCOM_extensions">Comparison with XUL/XPCOM extensions</a>
         <li><a href="<%=baseURL%>WebExtensions/Chrome_incompatibilities">Chrome incompatibilities</a></li>
+        <li><a href="<%=baseURL%>WebExtensions/Differences_between_desktop_and_Android_">Differences between desktop and Android</a></li>
```

Comment:
This doesn't seem to exist?
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Differences_between_desktop_and_Android_